doc/conf.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+from pkg_resources import get_distribution, DistributionNotFound
+import sys
+
+# sys.path.insert(0, os.path.abspath('.'))
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'source'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'ftrack_connect_pipeline_ftrack-connect-pipeline-3dsmax'
```

Comment:
Shouldn't this be: ftrack-connect-pipeline-3dsmax

Reply:
Docs will be attended in a future PR